### PR TITLE
Cleanup executeInContainer method

### PR DESCRIPTION
**Pull request by scoheb**

### CODE REVIEW

File: src/org/centos/pipeline/PipelineUtils.groovy [hunk 1]
```diff
@@ -632,22 +632,19 @@ def executeInContainer(String stageName, String containerName, String script) {
...
     //
     containerEnv = env.getEnvironment().collect { key, value -> return key+'='+value }
-    sh "mkdir -p " + stageName
+    sh "mkdir -p ${stageName}"
     try {
         withEnv(containerEnv) {
             container(containerName) {
-                sh 'pwd'
-                sh 'ls -l /tmp'
                 sh "cp -fv ${WORKSPACE}/fedora.keytab /home/fedora.keytab"
```
Comment (johnbieren) [inline src/org/centos/pipeline/PipelineUtils.groovy]: Could probably do an if stageName = rpmbuild then cp, but not sure that makes it cleaner =)

Comment (scoheb) [reply src/org/centos/pipeline/PipelineUtils.groovy]: we sure it is only needed by the rpmbuild container?

Comment (johnbieren) [reply src/org/centos/pipeline/PipelineUtils.groovy]: I am pretty sure it is only ever used in the build submission to koji, which is in the rpmbuild stage. The integration and functional tests don't use it. @p3ck none of the compose or booty sanity stages use it, correct?

Comment (p3ck) [reply src/org/centos/pipeline/PipelineUtils.groovy]: correct, nothing but rpmbuild uses the fedora keytab.

**Comment by scoheb** [comment]
[merge]

**Pull request merged**